### ISSUE TITLE
Intelligible Audio

### ISSUE THREAD

**Issue by PetrochukM** [r9y9/wavenet_vocoder#65]
Hi Everyone!

Training on LJSpeech with Mu-law, at around what step will the audio be more than noise? 

At 5k steps it sounds like white noise:
![image](https://user-images.githubusercontent.com/7424737/40143216-b48b6df0-590f-11e8-8dd2-148f7696e08a.png)

Training with a batch size of 24.

Thanks!

**Comment by PetrochukM** [comment]
Specifically, looks like ``save_sates`` with GTA sounds great at 10k steps but ``eval_model`` produces noise at 10k steps.

**Comment by PetrochukM** [comment]
![image](https://user-images.githubusercontent.com/7424737/40211230-51f54ba8-59fe-11e8-9d27-a193b9675d4d.png)

Following my PRs, fixing µ, getting rid of ``math.sqrt()``, removing weight normalization, etc convergence has significantly increased on the LJ speech dataset, see below:

![image](https://user-images.githubusercontent.com/7424737/40211256-84bb6e78-59fe-11e8-8141-a1a025de5826.png)

**Comment by r9y9** [comment]
What does the each line mean? What's the difference between two figures?

**Comment by r9y9** [comment]
What about the evaluation quality? From my experience I had many cases that there are quality drop while loss decrease.

**Comment by PetrochukM** [comment]
The first line was the default configuration with LJ speech Mu-Law. The second line, the learning rate was fixed to .001 and mu was set to 255. The third line, weight norm and math.sqrt was removed.

Until the third try, the results were white noise with eval

**Comment by PetrochukM** [comment]
Hi!

Experiment 1 deviations from master repo:

+ 24000 Sample Rate
+ No normalization
+ No ``amp_to_db`` stuff
+ No rescaling
+ Batch size 24 (8 GPUs)
+ Mulaw-quantize, mu = 256

Results:

- ``eval_model``: White noise
- ``synthesis``: Did not run

Experiment 2 deviations from master repo:

+ Remove RELu
+ Fix learning rate
+ Remove ``exponential_moving_average``

Results:

- ``eval_model``: White noise
- ``synthesis``: Human-like blabber

Experiment 3 deviations from master repo:

+ Remove weight normalization
+ Add 1 clip threshold
+ Remove ``math.sqrt``


Results:

- ``eval_model``: Sounds like the target with some noise
- ``synthesis``: Still training
